toolkit/scripts/toolchain/download_toolchain_rpm.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -e
+
+# Usage print
+function usage() {
+    echo "Usage: $0 --downloader-tool <downloader_tool> --rpm-name <rpm_name> --dst <dst_file> --log-base <log_base>" \
+    "--url-list <url_list> [--cert <cert>] [--key <key>] [--enforce-signatures] [--allowable-gpg-keys <allowable_gpg_keys>]"
+
+    echo "-t|--downloader-tool: Path to our go downloader tool"
+    echo "-r|--rpm-name: Name of the RPM to download"
+    echo "-d|--dst: Destination file path"
+    echo "-l|--log-base: Base path for log files. Each attempt will have a log file named <log_base>.<attempt_num>"
+    echo "--hydrate: Print alternate log message for hydrate operations"
+    echo "-h|--help: Print this help message"
+    echo "-u|--url-list: Space separated list of base URLs to attempt to download the RPM from. The URLs will be tried in order"
+    echo "           until the RPM is successfully downloaded. Full URL will be <url>/<rpm_name>"
+    echo "-c|--certificate: Optional path to a certificate file to use for the download"
+    echo "-k|--private-key: Optional path to a private key file to use for the download"
```

Comment:
does not match usage output above (--key vs --private-key, and --certificate vs --cert)

Reply:
Fixed